worlds/tits_the_3rd/client/memory_io.py

```diff
@@ -273,3 +274,102 @@ def read_flag(self, flag_number: int):
         data = self._read_bytes(flag_byte_offset, 1)
         flag_value = (data >> flag_bit) & 1
         return bool(flag_value)
+
+    def get_scena_offset(self, scena_id: bytes):
```

Comment:
Nit: lets add a return type to the definition + some documentation around the args / return:
`def get_scena_offset(self, scena_id: bytes) -> Optional[int]`

Reply:
Similar comment to other functions.